index.html

```diff
@@ -919,6 +919,93 @@ <h4>
   reader.scan({ recordType: "example.com:sp"});
       </pre>
     </aside>
+    <aside title="Push a smart poster message" class="example">
+      <p>
+        Smart poster defined in NFC Forum Smart Poster Record Type Definition
+        specification. It is an NDEF record that contains an <a>NDEF message</a>
+        as payload, which in turn may contain several records: a URI record that
+        refers to a content, and additional optional records for that content's
+        title, icon, size, type and possible action (e.g. send SMS, make call,
+        launch browser, or save for later, or open for editing, etc).
+      </p>
+      <pre class="highlight">
+        const writer = new NFCWriter();
+        let sp = new NDEFMessage({
+          records: [
+            {
+              recordType: "url",
+              data: "https://my.org/content/19911"
+            },
+            {
+              recordType: "t", // type, a local type to Sp
+              data: "image/gif"
+            },
+            {
+              recordType: "text",
+              data: "Funny dance"
+            },
+            {
+              recordType: "s",  // size, a local type to Sp
+              data: 4096  // byte size of the content at the URL above
+            },
+            {
+              recordType: "act",  // action, a local type to Sp
+              data: 0  // do the action, in this case open in the browser
+            },
+          ];
+        });
+        writer.push({ records: [
+          {
+            recordType: "smart-poster",
+            data: sp.records
```

Comment:
If you are just calling `sp.records` then why do we need to create the NDEFMessage object and not just an array of json records?

Reply:
I thought is was more clear as an example. You could have that Sp message from other source as well (e.g. previously read). But no problem, I can compact the example.